pkg/reconcilers/deploymentconfig.go

```diff
@@ -142,3 +142,16 @@ func DeploymentConfigEnvVarReconciler(desired, existing *appsv1.DeploymentConfig
 	}
 	return update
 }
+
+// GetConfigMutators returns generic mutators.
+// If an annotation to disable syncing is present,
+// replicas mutator is not included.
+func GetConfigMutators(annotations map[string]string, annotation string) []DCMutateFn {
```

Comment:
The name is confusing

It relies on `GenericOpts()` while it would be less coupled if list of mutators is passed. to the method